taskfiles/lint-yaml.yaml

```
@@ -0,0 +1,26 @@
+version: "3"
+
+tasks:
+  yaml:
+    aliases:
+      - "yaml-check"
+    desc: "Runs the YAML linters. Only checks for warnings and violations."
+    sources:
+      - "{{.G_LINT_VENV_CHECKSUM_FILE}}"
+      - "{{.ROOT_DIR}}/**/*.yaml"
+      - "{{.ROOT_DIR}}/**/*.yml"
+      - "{{.TASKFILE}}"
+      - exclude: "{{.ROOT_DIR}}/**/build/*"
+    dir: "{{.ROOT_DIR}}"
+    deps:
+      - "venv"
+    cmds:
+      - |-
+        . "{{.G_LINT_VENV_DIR}}/bin/activate"
+        find . \
+          \( -path '**/build' \) -prune -o \
```

```suggestion
          \( -path "*/build/*" \) -prune -o \
```

* As far as I can tell, `find` doesn't treat globstar (`**`) specially; so `*` should be equivalent to `**`.
* `*/build/*` more accurately matches the sources exclusion.
* I don't think single-quotes are necessary here, are they?

---

I'll test this out.

---

You're right. `find` uses shell pattern matching. The single quotes are necessary though and I don't think there's any point to changing them to double quotes.

---

Sorry, why are the single quotes necessary?

The reason I'm suggesting double quotes is to stay consistent with our style.

---

I see. The way you phrased it made it sound like you wanted to replace the single quotes with nothing, not double quotes.